lib/src/widget/nav_bar/widget/nav_bar_container.dart

```diff
@@ -205,8 +205,7 @@ class NavBarContainerState extends State<NavBarContainer>
 
     if (ignoreLast &&
         list.isNotEmpty &&
-        list.last.navBarConfig.id ==
-            currentNavBarConfigMixin?.navBarConfig.id) {
+        list.last.navBarConfig == currentNavBarConfigMixin?.navBarConfig) {
```

Comment:
💡 looks good to me, but please add a test, which covers this scenario

Reply:
Done